src/test/java/com/wikia/webdriver/testcases/mercurytests/NavigationTest.java

```diff
@@ -95,12 +91,28 @@ public void mercury_navigation_navigationElementsUserLoggedIn() {
   @Execute(asUser = User.ANONYMOUS)
   @Test(groups = "mercury_navigation_navigationElementsAnonymousUser")
   public void mercury_navigation_navigationElementsAnonymousUser() {
-    init();
+    Navigation navigation = this.getNavigation(MercurySubpages.MAIN_PAGE);
 
-    topBar.openNavigation();
     Assertion.assertTrue(navigation.isUserAvatarVisible());
     Assertion.assertFalse(navigation.isUserProfileLinkVisible());
     Assertion.assertFalse(navigation.isLogoutLinkVisible());
     Assertion.assertEquals(navigation.getNavigationHeaderText(), "Sign In | Register");
   }
+
+  @Test(groups = "mercury_navigation_exploreWikiNavigatesToWikiMainPage")
+  public void mercury_navigation_exploreWikiNavigatesToWikiMainPage() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)